robot_controllers/CMakeLists.txt

```diff
@@ -88,6 +123,7 @@ install(FILES robot_controllers.xml
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
 
-install(TARGETS robot_controllers LIBRARY
+install(TARGETS robot_controllers robot_controllers_python LIBRARY
```

Comment:
2 completely separate comments here:

1) not sure why I had the cmake this way -- I would move LIBRARY onto the next line (it's really part of the LIBRARY DESTINATION). At the same time, should add ARCHIVE DESTINATION in case someone later does a static build (see https://github.com/mikeferguson/robot_calibration/blob/indigo-devel/robot_calibration/CMakeLists.txt#L104 for an example)

2) This is actually the wrong place to put the robot_controllers_python library -- we put it in a special place above (inside the CATKIN_GLOBAL_PYTHON_DESTINATION in DEVEL_PREFIX) -- I think we would need to do something similar in the install rule -- but I can't find a good example of anyone releasing a boost::python module into a ROS deb